src/Traveler.js

```diff
@@ -0,0 +1,9 @@
+class Traveler {
+  constructor(traveler) {
```

Comment:
Good job passing an object as an argument and using the class constructor to parse out the values, looks really clean and easy to understand.